versioned_docs/version-8.0/components/connectors/out-of-the-box-connectors/rabbitmq.md

```diff
@@ -0,0 +1,125 @@
+---
+id: rabbitmq
+title: RabbitMQ Service Connector
+sidebar_label: RabbitMQ Connector
+description: Send messages to RabbitMQ from your BPMN process using the RabbitMQ Connector.
+---
+
+The **RabbitMQ Connector** allows you to connect your BPMN process with [RabbitMQ](https://www.rabbitmq.com/).
+
+## Prerequisites
+
+To use the **RabbitMQ Connector**, you need to have installed a RabbitMQ server and create the relevant [credentials](https://www.rabbitmq.com/passwords.html).
+It is highly recommended to use Camunda secrets to store credentials, so that you don't expose sensitive information directly from the process. See [this appendix entry](#how-do-i-store-secrets-for-my-connector) to learn more.
+
+:::note
+Ensure you enter the correct exchange name and routing key, as the **RabbitMQ Connector** can't throw an exception if they are incorrect.
+:::
+
+## Create a RabbitMQ Connector task
+
+To use the **RabbitMQ Connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu icon,
+
+![wrench-shape](../img/connectors-rabbitmq-wrench-shaped.png)
+
+or create a new Connector task by using the **Append Connector** context menu.
+
+![wrench-shape](../img/connectors-rabbitmq-append-connector.png)
+
+Follow our [guide to using Connectors](../use-connectors.md) to learn more.
+
+## Connecting to RabbitMQ and sending messages
+
+To connect to RabbitMQ, choose the required connection type in the **Authentication** section and complete the mandatory fields highlighted in red in the connector properties panel:
+
+![connectors-rabbitmq-red-properties](../img/connectors-rabbitmq-red-properties.png)
+
+:::note
+All the mandatory and non-mandatory fields depending on the authentication selection you choose are covered in the upcoming sections.
+:::
+
+## Authentication
+
+You can choose among the available RabbitMQ Connectors according to your authentication requirements.
```

Comment:
Are you referring here to the different authentication types? If that's the case, I would suggest something like:
"You can choose from the available authentication types of the RabbitMQ Connector depending on your authentication requirements."